code/modules/projectiles/projectile/ego_bullets/waw.dm

```diff
@@ -227,3 +227,65 @@
 		if(!ishuman(M) && !M.has_status_effect(/datum/status_effect/sunder_red))
 			new /obj/effect/temp_visual/cult/sparks(get_turf(M))
 			M.apply_status_effect(/datum/status_effect/sunder_red)
+
+/datum/status_effect/rendRedArmor
+	id = "rend red armor"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 30 //3 seconds
+	alert_type = null
+	var/original_armor
+
+/datum/status_effect/rendRedArmor/on_apply()
```

Comment:
Is rendarmor part of thunderbird or remnants of code from something else?

Reply:
I have reason to believe that this is deprecated code from exuviae E.G.O.; get rid of it.